packages/jaeger-ui-components/src/TraceTimelineViewer/SpanLinks.tsx

```diff
@@ -0,0 +1,80 @@
+import { css } from '@emotion/css';
+import React from 'react';
+import { SpanLinks } from 'src/types/links';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2, WithContextMenu, MenuGroup, MenuItem, Icon } from '@grafana/ui';
+
+interface SpanLinksProps {
+  links: SpanLinks;
+}
+
+const renderMenuItems = (links: SpanLinks) => {
+  return (
+    <>
+      {links.logLinks ? (
+        <MenuGroup label="Logs">
+          {links.logLinks.map((link, i) => (
+            <MenuItem
```

Comment:
I was surprised to not see a title here. Did some [digging](http://www.nysed.gov/webaccess/create-accessible-hypertext-links) and realised maybe it's not necessary. Plus it would probably be outside the scope of this pr.